.eslintrc.json

```diff
@@ -34,13 +34,14 @@
         "prettier/prettier": [
         "error",
         {
+            "endOfLine": "auto",
             "printWidth": 120,
             "singleQuote": true,
             "trailingComma" : "all",
             "arrow-parens": "avoid"
         }
     ],
-    "linebreak-style": ["error", "unix"],
+    "linebreak-style": ["error", "windows"],
```

Comment:
I won't suggest you change in lint file.. it's a pre define config and we need to follow it.